documentation/operators/src/nodes/setup.md

```diff
@@ -99,10 +106,46 @@ Some of the most useful flags and their explanation:
 - `--expose-crypto-hardware <true/false>`: Sets your crypto hardware info visibility on the network.
 ~~~
 
+### Terms & Conditions
+
+```admonish info
+From `nym-node` version `1.1.3` onward is required to accept [**Operators Terms & Conditions**]({{toc_page}}) in order to be part of the active set. Make sure to read them before you accept. 
+```
+
+There has been a long ongoing discussion whether and how to apply Terms and Conditions for Nym network operators, with an aim to stay aligned with the philosophy of Free Software and provide legal defense for both node operators and Nym developers. To understand better the reasoning behind this decision, you can listen to the first [Nym Operator Town Hall](https://www.youtube.com/live/7hwb8bAZIuc?si=3mQ2ed7AyUA1SsCp&t=915) introducing the T&Cs or to [Operator AMA with CEO Harry Halpin](https://youtu.be/yIN-zYQw0I0?feature=shared) from June 4th, 2024, explaining pros and cons of T&Cs implementation.
```

Comment:
I feel like we could also put a sort of 'tldr' of the reasoning behind the decision here as well as linking to the various calls. 

Also you have one `youtube` link and one `youtu.be` link: is this deliberate? Always a fan of `yewtu.be` > `youtube` but just wanted to check :)

Reply:
- [x] done